controllers/syncer/csi_node.go

```diff
@@ -104,11 +104,14 @@ func (s *csiNodeSyncer) SyncCSIDaemonsetFn(daemonSetRestartedKey string, daemonS
 		}
 	}
 
+	annotations := s.driver.GetAnnotations(daemonSetRestartedKey, daemonSetRestartedValue)
+	annotations["kubectl.kubernetes.io/default-logs-container"] = config.Product
```

Comment:
If `kubectl.kubernetes.io/default-logs-container` is a well-known kubernetes annotation, we can move the annotation key to constants.go file under `// Kubernetes built-in well-known constants`. Already have some defined there.